djhtml/modes.py

```diff
@@ -323,6 +331,22 @@ def create_token(self, raw_token, src):
         kind = "javascript"
         self.next_mode = self
 
+        if raw_token.strip() == "switch":
+            self.hard_indents = getattr(self, 'hard_indents', 0) + 1
+            self.opened_brackets = getattr(self, 'opened_brackets', 0)
+
```

Comment:
Not really fond of `getattr` for this usecase. Let's initialize `self.{hard_indents,opened_brackets}` to `0` so `getattr` isn't needed.

Also not really fond of using a counter instead of a proper stack, but I guess it is the simplest solution.

Reply:
Done in [commit](https://github.com/rtts/djhtml/pull/49/commits/390da4586377e43ed8693fea018c543f30d930d9).